app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+module Idv
+  module InPerson
+    class StateIdController < ApplicationController
+      include Idv::AvailabilityConcern
+      include IdvStepConcern
+
+      before_action :render_404_if_controller_not_enabled
+      before_action :redirect_unless_enrollment # confirm previous step is complete
+
+      def show
+        flow_session[:pii_from_user] ||= {}
+        analytics.idv_in_person_proofing_state_id_visited(**analytics_arguments)
+
+        render :show, locals: extra_view_variables
+      end
+
+      def extra_view_variables
+        {
+          form:,
+          pii:,
+          parsed_dob:,
+          updating_state_id: updating_state_id?,
+        }
+      end
+
+      # update Idv::DocumentCaptureController.step_info.next_steps to include
```

Comment:
i think so, but I thought it might be something to address in the pr that adds the update route since this pr doesn't allow you to navigate past the state id page